netbox/extras/models/customfields.py

```diff
@@ -177,12 +177,19 @@ class CustomField(CloningMixin, ExportTemplatesMixin, ChangeLoggedModel):
         blank=True,
         null=True
     )
-    ui_visibility = models.CharField(
+    ui_visible = models.CharField(
         max_length=50,
-        choices=CustomFieldVisibilityChoices,
-        default=CustomFieldVisibilityChoices.VISIBILITY_READ_WRITE,
-        verbose_name=_('UI visibility'),
-        help_text=_('Specifies the visibility of custom field in the UI')
+        choices=CustomFieldUIVisibleChoices,
+        default=CustomFieldUIVisibleChoices.ALWAYS,
+        verbose_name=_('UI visible'),
+        help_text=_('Specifies whether the custom field is displayed in the UI')
+    )
+    ui_editable = models.CharField(
```

Comment:
It looks like these two are bound together and it doesn't make sense to have something that is not visible but is editable?  If so I think there should be a check in the save to disallow these conflicting settings.

Reply:
I don't think either field is necessarily dependent on the other. A user may which to omit a custom field from display in the UI but still allow it to be viewable/editable "behind the scenes" for convenience when editing the object, for example.